vnext/local-cli/src/runWindows/utils/deploy.ts

```diff
@@ -54,7 +59,16 @@ function getAppPackage(options: RunWindowsOptions): string {
   const appPackageGlob = `${
     options.root
   }/windows/{*/AppPackages,AppPackages/*}/${packageFolder}`;
-  let appPackage = glob.sync(appPackageGlob)[0];
+  const globs = glob.sync(appPackageGlob);
+  let appPackage;
+  if (globs.length == 1 || !projectName) {
```

Comment:
'===' instead of '=='. As is will generate eslint warnings (see yarn lint in Extra Checks)

Reply:
This will also show up as a yellow squiggly if you have eslint configured in vs code. Would highly recommend you do that if you haven't.

Reply:
@NickGerleman I do have it configured, and it looks like it was showing up, but it's too easy to miss, esp on something as short as "==" :-(